### DISCUSSION TITLE
Twitter notifications

### DISCUSSION THREAD

**Discussion by gjj07** [jef/streetmerchant#3001] [Help]
Can anyone help me setup how the twitter notifications work? I have the developer account and I have the api keys but than when i entered them it said i needed a elevated level of access which i applied for and received. Do i need to turn on the OAuth 2.0 or Oauth 1.0? if so what do you put for the call back url? I'm so confused any help is greatly appreciated.

**Comment by jef** [comment]
could be related to https://github.com/jef/streetmerchant/issues/3007. i will look at this some point in the near future :)
